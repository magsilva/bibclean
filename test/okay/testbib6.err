%% testbib6.bib:7:Unexpected value in ``month = MAR # "- " # APR''.
%% testbib6.bib:8:Unexpected value in ``month = MAR # "-" # APR''.
%% testbib6.bib:9:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:10:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:12:Unexpected value in ``month = MAR # "-- " # APR''.
%% testbib6.bib:14:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:22:Unexpected value in ``month = MAR # " \endash{} " # APR''.
%% testbib6.bib:23:Unexpected value in ``month = MAR # "\endash{} " # APR''.
%% testbib6.bib:24:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:25:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:27:Unexpected value in ``month = MAR # " \endash " # APR''.
%% testbib6.bib:29:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:37:Unexpected value in ``month = mar # "- " # APR''.
%% testbib6.bib:38:Unexpected value in ``month = mar # "-" # APR''.
%% testbib6.bib:39:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:40:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:42:Unexpected value in ``month = mar # "-- " # APR''.
%% testbib6.bib:44:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:52:Unexpected value in ``month = mar # " \endash{} " # APR''.
%% testbib6.bib:53:Unexpected value in ``month = mar # "\endash{} " # APR''.
%% testbib6.bib:54:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:55:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:57:Unexpected value in ``month = mar # " \endash " # APR''.
%% testbib6.bib:59:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:67:Unexpected value in ``month = MAR # "- " # apr''.
%% testbib6.bib:68:Unexpected value in ``month = MAR # "-" # apr''.
%% testbib6.bib:69:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:70:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:72:Unexpected value in ``month = MAR # "-- " # apr''.
%% testbib6.bib:74:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:82:Unexpected value in ``month = MAR # " \endash{} " # apr''.
%% testbib6.bib:83:Unexpected value in ``month = MAR # "\endash{} " # apr''.
%% testbib6.bib:84:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:85:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:87:Unexpected value in ``month = MAR # " \endash " # apr''.
%% testbib6.bib:89:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:97:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:98:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:99:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:100:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:102:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:104:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:112:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:113:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:114:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:115:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:117:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:119:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:127:Unexpected value in ``month = MAR # "- " # APR''.
%% testbib6.bib:128:Unexpected value in ``month = MAR # "-" # APR''.
%% testbib6.bib:129:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:130:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:132:Unexpected value in ``month = MAR # "-- " # APR''.
%% testbib6.bib:134:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:142:Unexpected value in ``month = MAR # " \endash{} " # APR''.
%% testbib6.bib:143:Unexpected value in ``month = MAR # "\endash{} " # APR''.
%% testbib6.bib:144:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:145:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:147:Unexpected value in ``month = MAR # " \endash " # APR''.
%% testbib6.bib:149:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:157:Unexpected value in ``month = mar # "- " # APR''.
%% testbib6.bib:158:Unexpected value in ``month = mar # "-" # APR''.
%% testbib6.bib:159:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:160:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:162:Unexpected value in ``month = mar # "-- " # APR''.
%% testbib6.bib:164:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:172:Unexpected value in ``month = mar # " \endash{} " # APR''.
%% testbib6.bib:173:Unexpected value in ``month = mar # "\endash{} " # APR''.
%% testbib6.bib:174:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:175:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:177:Unexpected value in ``month = mar # " \endash " # APR''.
%% testbib6.bib:179:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:187:Unexpected value in ``month = MAR # "- " # apr''.
%% testbib6.bib:188:Unexpected value in ``month = MAR # "-" # apr''.
%% testbib6.bib:189:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:190:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:192:Unexpected value in ``month = MAR # "-- " # apr''.
%% testbib6.bib:194:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:202:Unexpected value in ``month = MAR # " \endash{} " # apr''.
%% testbib6.bib:203:Unexpected value in ``month = MAR # "\endash{} " # apr''.
%% testbib6.bib:204:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:205:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:207:Unexpected value in ``month = MAR # " \endash " # apr''.
%% testbib6.bib:209:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:217:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:218:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:219:Unexpected value in ``month = mar # "- " # apr''.
%% testbib6.bib:220:Unexpected value in ``month = mar # "-" # apr''.
%% testbib6.bib:222:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:224:Unexpected value in ``month = mar # "-- " # apr''.
%% testbib6.bib:232:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:233:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:234:Unexpected value in ``month = mar # " \endash{} " # apr''.
%% testbib6.bib:235:Unexpected value in ``month = mar # "\endash{} " # apr''.
%% testbib6.bib:237:Unexpected value in ``month = mar # " \endash " # apr''.
%% testbib6.bib:239:Unexpected value in ``month = mar # " \endash " # apr''.
