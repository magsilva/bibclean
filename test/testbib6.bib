@Article{x,
	MONTH = MAR # "\slash " # APR,
	MONTH = MAR # "\slash" # APR,
	MONTH = mar # "\slash " # apr,
	month = mar # "\slash" # apr,

	MONTH = MAR # "- " # APR,
	MONTH = MAR # "-" # APR,
	MONTH = mar # "- " # apr,
	month = mar # "-" # apr,

	MONTH = MAR # "-- " # APR,
	MONTH = MAR # "--" # APR,
	MONTH = mar # "-- " # apr,
	month = mar # "--" # apr,

	MONTH = MAR # " -- " # APR,
	MONTH = MAR # "--" # APR,
	MONTH = mar # " -- " # apr,
	month = mar # "--" # apr,

	MONTH = MAR # " \endash{}  " # APR,
	MONTH = MAR # "\endash{} " # APR,
	MONTH = mar # " \endash{}  " # apr,
	month = mar # "\endash{} " # apr,

	MONTH = MAR # " \endash  " # APR,
	MONTH = MAR # "\endash " # APR,
	MONTH = mar # " \endash  " # apr,
	month = mar # "\endash " # apr,

	MONTH = "March" # "\slash " # APR,
	MONTH = "March" # "\slash" # APR,
	MONTH = mar # "\slash " # apr,
	month = mar # "\slash" # apr,

	MONTH = "March" # "- " # APR,
	MONTH = "March" # "-" # APR,
	MONTH = mar # "- " # apr,
	month = mar # "-" # apr,

	MONTH = "March" # "-- " # APR,
	MONTH = "March" # "--" # APR,
	MONTH = mar # "-- " # apr,
	month = mar # "--" # apr,

	MONTH = "March" # " -- " # APR,
	MONTH = "March" # "--" # APR,
	MONTH = mar # " -- " # apr,
	month = mar # "--" # apr,

	MONTH = "March" # " \endash{}  " # APR,
	MONTH = "March" # "\endash{} " # APR,
	MONTH = mar # " \endash{}  " # apr,
	month = mar # "\endash{} " # apr,

	MONTH = "March" # " \endash  " # APR,
	MONTH = "March" # "\endash " # APR,
	MONTH = mar # " \endash  " # apr,
	month = mar # "\endash " # apr,

	MONTH = MAR # "\slash " # "April",
	MONTH = MAR # "\slash" # "April",
	MONTH = mar # "\slash " # "April",
	month = mar # "\slash" # "April",

	MONTH = MAR # "- " # "April",
	MONTH = MAR # "-" # "April",
	MONTH = mar # "- " # "April",
	month = mar # "-" # "April",

	MONTH = MAR # "-- " # "April",
	MONTH = MAR # "--" # "April",
	MONTH = mar # "-- " # "April",
	month = mar # "--" # "April",

	MONTH = MAR # " -- " # "April",
	MONTH = MAR # "--" # "April",
	MONTH = mar # " -- " # "April",
	month = mar # "--" # "April",

	MONTH = MAR # " \endash{}  " # "April",
	MONTH = MAR # "\endash{} " # "April",
	MONTH = mar # " \endash{}  " # "April",
	month = mar # "\endash{} " # "April",

	MONTH = MAR # " \endash  " # "April",
	MONTH = MAR # "\endash " # "April",
	MONTH = mar # " \endash  " # "April",
	month = mar # "\endash " # "April",

	MONTH = "March" # "\slash " # "April",
	MONTH = "March" # "\slash" # "April",
	MONTH = mar # "\slash " # "April",
	month = mar # "\slash" # "April",

	MONTH = "March" # "- " # "April",
	MONTH = "March" # "-" # "April",
	MONTH = mar # "- " # "April",
	month = mar # "-" # "April",

	MONTH = "March" # "-- " # "April",
	MONTH = "March" # "--" # "April",
	MONTH = mar # "-- " # "April",
	month = mar # "--" # "April",

	MONTH = "March" # " -- " # "April",
	MONTH = "March" # "--" # "April",
	MONTH = mar # " -- " # "April",
	month = mar # "--" # "April",

	MONTH = "March" # " \endash{}  " # "April",
	MONTH = "March" # "\endash{} " # "April",
	MONTH = mar # " \endash{}  " # "April",
	month = mar # "\endash{} " # "April",

	MONTH = "March" # " \endash  " # "April",
	MONTH = "March" # "\endash " # "April",
	MONTH = mar # " \endash  " # apr,
	month = mar # "\endash " # apr,

	MONTH = MAR # "" # "" # "" # "\slash " # "" # "" # "" # APR,
	MONTH = MAR # "" # "" # "" # "\slash" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # "\slash " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\slash" # "" # "" # "" # apr,

	MONTH = MAR # "" # "" # "" # "- " # "" # "" # "" # APR,
	MONTH = MAR # "" # "" # "" # "-" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # "- " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "-" # "" # "" # "" # apr,

	MONTH = MAR # "" # "" # "" # "-- " # "" # "" # "" # APR,
	MONTH = MAR # "" # "" # "" # "--" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # "-- " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "--" # "" # "" # "" # apr,

	MONTH = MAR # "" # "" # "" # " -- " # "" # "" # "" # APR,
	MONTH = MAR # "" # "" # "" # "--" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # " -- " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "--" # "" # "" # "" # apr,

	MONTH = MAR # "" # "" # "" # " \endash{}  " # "" # "" # "" # APR,
	MONTH = MAR # "" # "" # "" # "\endash{} " # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # " \endash{}  " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\endash{} " # "" # "" # "" # apr,

	MONTH = MAR # "" # "" # "" # " \endash  " # "" # "" # "" # APR,
	MONTH = MAR # "" # "" # "" # "\endash " # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # " \endash  " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\endash " # "" # "" # "" # apr,

	MONTH = "March" # "" # "" # "" # "\slash " # "" # "" # "" # APR,
	MONTH = "March" # "" # "" # "" # "\slash" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # "\slash " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\slash" # "" # "" # "" # apr,

	MONTH = "March" # "" # "" # "" # "- " # "" # "" # "" # APR,
	MONTH = "March" # "" # "" # "" # "-" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # "- " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "-" # "" # "" # "" # apr,

	MONTH = "March" # "" # "" # "" # "-- " # "" # "" # "" # APR,
	MONTH = "March" # "" # "" # "" # "--" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # "-- " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "--" # "" # "" # "" # apr,

	MONTH = "March" # "" # "" # "" # " -- " # "" # "" # "" # APR,
	MONTH = "March" # "" # "" # "" # "--" # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # " -- " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "--" # "" # "" # "" # apr,

	MONTH = "March" # "" # "" # "" # " \endash{}  " # "" # "" # "" # APR,
	MONTH = "March" # "" # "" # "" # "\endash{} " # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # " \endash{}  " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\endash{} " # "" # "" # "" # apr,

	MONTH = "March" # "" # "" # "" # " \endash  " # "" # "" # "" # APR,
	MONTH = "March" # "" # "" # "" # "\endash " # "" # "" # "" # APR,
	MONTH = mar # "" # "" # "" # " \endash  " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\endash " # "" # "" # "" # apr,

	MONTH = MAR # "" # "" # "" # "\slash " # "" # "" # "" # "April",
	MONTH = MAR # "" # "" # "" # "\slash" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # "\slash " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "\slash" # "" # "" # "" # "April",

	MONTH = MAR # "" # "" # "" # "- " # "" # "" # "" # "April",
	MONTH = MAR # "" # "" # "" # "-" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # "- " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "-" # "" # "" # "" # "April",

	MONTH = MAR # "" # "" # "" # "-- " # "" # "" # "" # "April",
	MONTH = MAR # "" # "" # "" # "--" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # "-- " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "--" # "" # "" # "" # "April",

	MONTH = MAR # "" # "" # "" # " -- " # "" # "" # "" # "April",
	MONTH = MAR # "" # "" # "" # "--" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # " -- " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "--" # "" # "" # "" # "April",

	MONTH = MAR # "" # "" # "" # " \endash{}  " # "" # "" # "" # "April",
	MONTH = MAR # "" # "" # "" # "\endash{} " # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # " \endash{}  " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "\endash{} " # "" # "" # "" # "April",

	MONTH = MAR # "" # "" # "" # " \endash  " # "" # "" # "" # "April",
	MONTH = MAR # "" # "" # "" # "\endash " # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # " \endash  " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "\endash " # "" # "" # "" # "April",

	MONTH = "March" # "" # "" # "" # "\slash " # "" # "" # "" # "April",
	MONTH = "March" # "" # "" # "" # "\slash" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # "\slash " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "\slash" # "" # "" # "" # "April",

	MONTH = "March" # "" # "" # "" # "- " # "" # "" # "" # "April",
	MONTH = "March" # "" # "" # "" # "-" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # "- " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "-" # "" # "" # "" # "April",

	MONTH = "March" # "" # "" # "" # "-- " # "" # "" # "" # "April",
	MONTH = "March" # "" # "" # "" # "--" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # "-- " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "--" # "" # "" # "" # "April",

	MONTH = "March" # "" # "" # "" # " -- " # "" # "" # "" # "April",
	MONTH = "March" # "" # "" # "" # "--" # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # " -- " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "--" # "" # "" # "" # "April",

	MONTH = "March" # "" # "" # "" # " \endash{}  " # "" # "" # "" # "April",
	MONTH = "March" # "" # "" # "" # "\endash{} " # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # " \endash{}  " # "" # "" # "" # "April",
	month = mar # "" # "" # "" # "\endash{} " # "" # "" # "" # "April",

	MONTH = "March" # "" # "" # "" # " \endash  " # "" # "" # "" # "April",
	MONTH = "March" # "" # "" # "" # "\endash " # "" # "" # "" # "April",
	MONTH = mar # "" # "" # "" # " \endash  " # "" # "" # "" # apr,
	month = mar # "" # "" # "" # "\endash " # "" # "" # "" # apr,
}
