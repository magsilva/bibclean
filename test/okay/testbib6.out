@Article{x,
  month =        MAR # "\slash " # APR,
  month =        MAR # "\slash" # APR,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        MAR # "- " # APR,
  month =        MAR # "-" # APR,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        MAR # "-- " # APR,
  month =        MAR # "--" # APR,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " -- " # APR,
  month =        MAR # "--" # APR,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " \endash{} " # APR,
  month =        MAR # "\endash{} " # APR,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        MAR # " \endash " # APR,
  month =        MAR # "\endash " # APR,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        mar # "\slash " # APR,
  month =        mar # "\slash" # APR,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        mar # "- " # APR,
  month =        mar # "-" # APR,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        mar # "-- " # APR,
  month =        mar # "--" # APR,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " -- " # APR,
  month =        mar # "--" # APR,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " \endash{} " # APR,
  month =        mar # "\endash{} " # APR,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        mar # " \endash " # APR,
  month =        mar # "\endash " # APR,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        MAR # "\slash " # apr,
  month =        MAR # "\slash" # apr,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        MAR # "- " # apr,
  month =        MAR # "-" # apr,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        MAR # "-- " # apr,
  month =        MAR # "--" # apr,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " -- " # apr,
  month =        MAR # "--" # apr,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " \endash{} " # apr,
  month =        MAR # "\endash{} " # apr,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        MAR # " \endash " # apr,
  month =        MAR # "\endash " # apr,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        MAR # "\slash " # APR,
  month =        MAR # "\slash" # APR,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        MAR # "- " # APR,
  month =        MAR # "-" # APR,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        MAR # "-- " # APR,
  month =        MAR # "--" # APR,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " -- " # APR,
  month =        MAR # "--" # APR,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " \endash{} " # APR,
  month =        MAR # "\endash{} " # APR,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        MAR # " \endash " # APR,
  month =        MAR # "\endash " # APR,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        mar # "\slash " # APR,
  month =        mar # "\slash" # APR,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        mar # "- " # APR,
  month =        mar # "-" # APR,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        mar # "-- " # APR,
  month =        mar # "--" # APR,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " -- " # APR,
  month =        mar # "--" # APR,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " \endash{} " # APR,
  month =        mar # "\endash{} " # APR,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        mar # " \endash " # APR,
  month =        mar # "\endash " # APR,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        MAR # "\slash " # apr,
  month =        MAR # "\slash" # apr,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        MAR # "- " # apr,
  month =        MAR # "-" # apr,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        MAR # "-- " # apr,
  month =        MAR # "--" # apr,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " -- " # apr,
  month =        MAR # "--" # apr,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        MAR # " \endash{} " # apr,
  month =        MAR # "\endash{} " # apr,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        MAR # " \endash " # apr,
  month =        MAR # "\endash " # apr,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        mar # "\slash " # apr,
  month =        mar # "\slash" # apr,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        mar # "- " # apr,
  month =        mar # "-" # apr,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        mar # "-- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " -- " # apr,
  month =        mar # "--" # apr,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        mar # " \endash{} " # apr,
  month =        mar # "\endash{} " # apr,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
  month =        mar # " \endash " # apr,
  month =        mar # "\endash " # apr,
}
